src/controllers/SizeController.ts

```diff
@@ -63,250 +63,205 @@ export class SizeController
+                }
+
+                this.layout.content.children.forEach((child) =>
+                {
+                    if (child === firstChild)
                     {
-                        childrenWidth += firstChild.width;
+                        // skip first element as it was already added
+                        return;
                     }
 
-                    this.layout.content.children.forEach((child) =>
+                    if (child.layout && child.layout.style.display !== 'block')
                     {
-                        if (child === firstChild)
+                        if (child.layout.style.position)
                         {
-                            // skip first element as it was already added
                             return;
                         }
 
-                        if (child.layout && child.layout.style.display !== 'block')
-                        {
-                            if (child.layout.style.position)
-                            {
-                                return;
-                            }
-
-                            childrenWidth += child.width + child.layout.style.marginLeft;
-                        }
-                        else if (child instanceof Container && child.width)
-                        {
-                            childrenWidth += child.width;
-                        }
-                    });
-
-                    // height is basing on content height
-                    finalWidth = childrenWidth + paddingLeft + paddingRight;
-
-                    if (isItJustAText(this.layout))
+                        childrenWidth += child.width + child.layout.style.marginLeft;
+                    }
+                    else if (child instanceof Container && child.width)
                     {
-                        finalWidth = this.innerText?.width + paddingLeft + paddingRight;
+                        childrenWidth += child.width;
                     }
+                });
 
-                    break;
+                // height is basing on content height
+                finalWidth = childrenWidth + paddingLeft + paddingRight;
+                break;
 
-                case 'parentSize':
-                default:
-                    // resize to parent width
-                    finalWidth = this.parentWidth;
+            case 'parentSize':
+                // resize to parent width
+                finalWidth = this.parentWidth;
 
-                    if (isItJustAText(this.layout))
-                    {
-                        this.innerText.style.wordWrap = true;
-                        this.innerText.style.wordWrapWidth
-                            = parentWidth - paddingLeft - paddingRight;
-                    }
+                break;
 
-                    break;
-            }
+            case 'static':
+            default:
+                finalWidth = getNumber(width, this.parentWidth);
+                break;
         }
```

Comment:
Those are just linting as the wrap condition was removed, the code here is the same, it's only `else` block is moved to
```
case 'static':
```